fe/fe-core/src/main/java/com/starrocks/catalog/TabletInvertedIndex.java

```diff
@@ -432,19 +433,28 @@ public void addTablet(long tabletId, TabletMeta tabletMeta) {
     }
 
     @VisibleForTesting
-    public Set<Long> getForceDeleteTablets() {
+    public Map<Long, Set<Long>> getForceDeleteTablets() {
         return forceDeleteTablets;
     }
 
-    public boolean tabletForceDelete(long tabletId) {
-        return forceDeleteTablets.contains(tabletId);
+    public boolean tabletForceDelete(long tabletId, long backendId) {
+        if (forceDeleteTablets.containsKey(tabletId)) {
+            return forceDeleteTablets.get(tabletId).contains(backendId);
+        }
+        return false;
     }
 
-    public void markTabletForceDelete(long tabletId) {
-        forceDeleteTablets.add(tabletId);
+    public void markTabletForceDelete(long tabletId, Set<Long> backendIds) {
+        forceDeleteTablets.add(tabletId, backendIds);
     }
     
-    public void eraseTabletForceDelete(long tabletId) {
+    public void eraseTabletForceDelete(long tabletId, long backendId) {
```

Comment:
In some situation, the tablet may not be dropped in BE because of high load. So the drop task will be send repeatedly until the replica is deleted on BE.

Reply:
There is a simple way: just store the tablet in a key expired map, and set the expire time to 10 times of the interval of tablet report(1 min).